Sources/Swift/Integrations/SessionReplay/SentryReplayOptions.swift

```diff
@@ -48,6 +48,11 @@ public class SentryReplayOptions: NSObject {
      */
     let sessionSegmentDuration = TimeInterval(5)
     
+    /**
+     * The maximum duration of a replay session.
+     */
+    let maximumDuration = TimeInterval(3_600)
```

Comment:
`l`: Really 60 minutes? How can this be so high?

Reply:
@bruno-garcia ??